README.adoc

```diff
@@ -2,6 +2,17 @@
 
 :repo-master: https://github.com/aerospike-community/spring-data-aerospike-starters/blob/master
 
+== Versions compatibility table
+
+|===
+|`spring-data-aerospike-starters` |`spring-data-aerospike` |`aerospike-client` |`aerospike-reactor-client`
```

Comment:
From the user perspective I think it will be difficult to use 2 tables. There is already one in https://github.com/aerospike-community/spring-data-aerospike#spring-data-aerospike-compatibility
Maybe we can move compatibility matrix into the separate project? In that project we can also setup automation that verifies that the projects of different versions actually work together...

Reply:
Not sure, since the compatibility tables are different for each project (yes it uses the same dependencies but each project can configure a different combination I guess).

I felt its important to specify it here since we are releasing a new starters version in Aug 2021 that uses clients from Sep 2020 (4.4.x).

I think we can merge it and think it through - if we come up with a better solution (maybe something similar to what you proposed, automating the version verification is a great idea) we can just implement it later since modifying the README files doesn't require releases/breaks. 
What do you think?

Reply:
ok